DuckDuckGo/Statistics/GeneralPixel.swift

```diff
@@ -1442,4 +1446,30 @@ enum GeneralPixel: PixelKitEventV2 {
         }
 
     }
+
+    public enum CompileTimeBucketAggregation: String, CustomStringConvertible {
+
+        public var description: String { rawValue }
+
+        case lessThan1 = "1"
+        case lessThan2 = "2"
+        case lessThan5 = "5"
+        case lessThan10 = "10"
+        case more
```

Comment:
I think we need a bit different buckets: on my mac it varies around 1.8-2.1s which puts me in either bucket 2 or 5, so we are losing some resolution here. I'd suggest: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20+  

We can tweak this if needed as well later. And remember about Privacy Triage for pixels :)